fdb-extensions/src/main/java/com/apple/foundationdb/synchronizedsession/SynchronizedSession.java

```diff
@@ -188,6 +188,24 @@ public UUID getSessionId() {
         });
     }
 
+    /**
+     * End any session by releasing the lock no matter this session holds the lock or not.
```

Comment:
```suggestion
     * End any session by releasing the lock no matter whether this session holds the lock or not.
```

Reply:
Should it be "End any session" here, or "End this session"?

Reply:
Maybe it is worth noting a bit of the details about how this works. For example, highlighting that it only takes place when this transaction is committed and will only be enforced when the other process holding the lease goes to check or renew the lease.